Bag of words/Test_3.py

```diff
@@ -1,22 +1,45 @@
-import time
-import json
-import tornado
+
+
+import tornado  
+
+import os
 from sklearn.feature_extraction.text import TfidfVectorizer
 from sklearn.metrics.pairwise import cosine_similarity
 import numpy as np
-
 vectorizer=TfidfVectorizer()
-text=open('newfile.txt')
-corpus = text.readlines()
-text.close()
+file_list = [f for f in os.listdir('.') if os.path.isfile(os.path.join('.', f)) and f.endswith("csv")]
+
+fcmap={}
+columnf= open("columns","w+")
+for f in file_list:
+    csvf = open(f)
+    columnl=csvf.__next__().split(",")
+    fcmap[f]=columnl
+    for word in columnl:
+        columnf.write(word+"\n")
+    csvf.close()
+
+columnf.close()
+columnf=open("columns")
+corpus = columnf.readlines()
```

Comment:
Use pandas to get column names / open and read CSV files. https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv